app/assets/javascripts/rails_admin/custom/ui.js

```diff
@@ -11,4 +11,7 @@ $(function() {
       });
     }
   });
-});
+}
+
+$(document).ready(makeHasManyAssociationTabsDraggable);
```

Comment:
`$(makeHasManyAssociationTabsDraggable);` would be enough here. See https://api.jquery.com/ready/